### PR TITLE
Rename dos locations

### PR DESCRIPTION
**Pull request by TheDoubleK**
Migrates outcomes and recruitment locations to plain old `locations`.

### CODE REVIEW

File: scripts/oneoff/rename-dos-locations.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            update["recruitLocations"] = None
+
+        if update:
+            counter += 1
+
+            update_reason = "rename locations: https://www.pivotaltracker.com/story/show/113271913"
+            if dry_run:
+                print("{} {}".format(draft["id"], update))
+            else:
+                client.update_draft_service(draft["id"], update, update_reason)
```
Comment (allait) [inline scripts/oneoff/rename-dos-locations.py]: None of the apiclient methods accept `update_reason` anymore. While we could send anything instead of the user name the value ends up being stored in the audit event `.user` field, so I don't think we should be doing this.

Comment (robyoung) [inline scripts/oneoff/rename-dos-locations.py]: I think a dummy user name here is less useful than something that actually identifies why the change was made.

Comment (TheDoubleK) [inline scripts/oneoff/rename-dos-locations.py]: Yes, this was deliberate, so that the audit events would make clear why the change was made - who ran the script to make the update is irrelevant.

Comment (allait) [inline scripts/oneoff/rename-dos-locations.py]: I'm still not sure about this. I know the user field is not strictly machine readable since we put both user emails and `whoami` there, but it's still `user`, so I think you could expect to see a name or a reference there. Reason text would look like the fields got mixed up. Putting a script name instead of reason might be a good middle ground.

I've just remembered another problem with draft updates though – [we actually look up the last event user in the API](https://github.com/alphagov/digitalmarketplace-api/blob/master/app/main/views/drafts.py#L171). This is then [used by the supplier frontend to show "Last edited by"](https://github.com/alphagov/digitalmarketplace-supplier-frontend/blob/master/app/templates/services/service_submission.html#L76). So if we do this without setting the user to an existing user email, it will show `Last edited <date> by `. I'm not sure what's the best approach here. We could use the admin account email address to make the message look right, but then we're not actually making any visible changes to the service data.

**Comment by allait** [comment]
Was run in production :+1:
